Functions.Templates/Templates/SqlOutputBinding-JavaScript/function.json

```diff
@@ -0,0 +1,22 @@
+{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "name": "req",
+      "type": "httpTrigger",
+      "direction": "in",
+      "methods": [
+        "get",
```

Comment:
Remove if we're not supporting GET in the sample

Reply:
we do support get, just that I haven't added a sample of output binding with get.

Reply:
I'm saying that the sample doesn't support GET requests so we shouldn't have that included in the methods. Users are free to add it later on of course if they want, but we shouldn't be giving them a partially-completed function. 

(and then yes, later on when we fix this to support both GET and POST we can update it then)

Reply:
ping, please remove GET as the sample doesn't currently support it. We can add it in later if we update the template to support both.